### DISCUSSION TITLE
I get an http 500 error after applying an a-z sort on the title field in the pnp search results web part....

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3470] [Q&A]
Hi,

I configured the sort under page 1 of 4 for the pnp search results web part properties using "Edit sort settings" then setting Title to be the default sort Ascending but i get an error....

This is the error:

[HTTP]:500 - [CorrelationId]:8f8604a1-d090-8000-0131-7531aee4bab9 [Version]:16.0.0.24510

I looked in Edge Dev Tools - Network and have an http 500, 404 and two 403 errors...

500 = This XML file does not appear to have any style information associated with it. The document tree is shown below., url = _api/search/postquery
404 = irrelevant, it is to do with my profile picture
403 = IF clicked this is opened: https://substrate.office.com/KnowledgeGraph/api/v1.0/Annotation with another error
404 = This XML file does not appear to have any style information associated with it. The document tree is shown below, url = /_api/SPHSite/Details

- All i want is to sort the search results alphabetically - can anyone suggest how to do this?

**Comment by kasperbolarsen** [comment]
Unless you have done some custom mapping, the Title field is actually not refinable nor sortable.
You will have to map the crawled properties of the Title propertie to a RefinableString for it to be refinable and sortable. 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/a5fc5b6d-446c-4ca2-aea6-c303f4159561)

**Reply by steveh2023** [reply]
Hi,

Thanks very much for the info, after searching online all morning I was wondering if that might be it...do you know of any info/guide to do this please?

**Reply by kasperbolarsen** [reply]
I would start looking at one of the youtube videos when searching for "sharepoint managed property"

**Reply by kasperbolarsen** [reply]
or perhaps this blog post can guide you, https://www.m365thinking.com/post/fix-one-of-the-problems-in-sharepoint-search-make-title-sortable

**Reply by steveh2023** [reply]
Thanks - that 2nd link looks really useful - I will try that later today...
